src/dom/README.md

```diff
@@ -267,94 +267,111 @@ assert.isFalse(ionString === ionString2);
 
 ### `dom.Value` equivalence
 
-dom.Value object equivalence is implemented by two methods ionEquals(compares two dom.Value objects based on strict or non-strict comparison type) and equals(compares a dom.Value object with a corresponding JS object).
+`dom.Value` provides two ways to test equality:
+1. `ionEquals`, which tests two `dom.Value` instances for equivalence according to the Ion data model.
+2. `equals`, which tests for value-based equality and can be used to compare a `dom.Value` with a corresponding JS object.
 
-#### `ionEquals`:
+#### `ionEquals`
 
-This method compares a dom.Value object with another dom.Value object with either strict or non-strict comparison type. 
+This method compares a `dom.Value` object with another `dom.Value` object with either strict or non-strict comparison type. 
 
 *Strict equivalence* refers to Ion data model equivalence as defined in Ion Equivalence (https://www.javadoc.io/doc/com.amazon.ion/ion-java/latest/com/amazon/ion/util/Equivalence.html) and by Ion Specification (http://amzn.github.io/ion-docs/docs/spec.html)
-*Structural or non-strict equivalence* follows the same rules as strict equivalence,except that
+*Structural* (or *non-strict*) *equivalence* follows the same rules as strict equivalence,except that:
+* Annotations are not considered, and
+* Timestamps that represent the same instant in time are always considered equivalent.
 
-* 1. Annotations are not considered, and
-* 2. Timestamps that represent the same instant in time are always considered equivalent.
+Optional parameters that can be used with this method:
 
-#### `equals`:
+| option                      | description                                                                    |
+| --------------------------- | :------------------------------------------------------------------------------|
+|`epsilon`                    | used by `Float` for an equality with given epsilon precision. (Default: `null`)|
+|`ignoreAnnotations`          | specifies whether to ignore annotations or not for equality. (Default: `false`)|
+|`ignoreTimestampPrecision`   | specifies whether to ignore timestamp local offset and precision or not for equality. (Default: `false`)|
 
-This method compares a dom.Value object with a corresponding JS object. 
+#### `equals`
 
-#### Integer Example:
+This method compares a `dom.Value` object with a corresponding JS object as defined in [Ion Data Types section](#iondom-data-types). 
+If the other object is a `dom.Value` instance then this comparison checks for structural (or "non-strict") equivalence.
+
+_NOTE:_ 
+
+* This method also provides an optional parameter `epsilon` for `Float` type comparison.
+* This method allows coerce numeric types comparison for `Decimal` and `Float` values:
+    * Comparison of `Float` with `Decimal` or `number` type is allowed.
+    * Comparison of `Decimal` with `Float` or `number` type is allowed.
```

Comment:
Can this be simplified to:
```suggestion
* This method allows `Decimal`, `Float`, and `number` values to be compared to one another.
```
?

Does it _not_ allow `Integer` values to be compared to the others?

Reply:
No it only allows `number` and `JSBI` with `Integer`.

Reply:
That's surprising to me--I expected value-based `equals` to allow `Integer`-to-`Float`/`Decimal` comparisons. I've opened #696 to track this.